R/FDAClassifTask.R

```diff
@@ -0,0 +1,46 @@
+#' @title Functional data classification task.
+#'
+#' @description
+#' Create a functional data classification task. This means that some features 
+#' in the task will be so-called functional covariates / functional features, 
+#' measured on a grid or time scale. Different covariates might come from
+#' different sensors for example.
+#'
+#' @inheritParams Task
+#' @template arg_fdatask_pars
+#' @return [\code{\link{FDAClassifTask}}].
+#' @export
+#' @examples
+#' dat = data.frame(matrix(rnorm(20), nrow = 2))
+#' dat$target = as.factor(c(0,1))
+#' # X1 to X5 is functional covariate 1 and X6 to X10 functional covariate 2
```

Comment:
We need to be consistent. Either it's covariate everywhere or feature everywhere. Don't mix or it gets confusing. In mlr we only use feature